packages/react-instantsearch-core/src/core/indexUtils.js

```diff
@@ -1,4 +1,4 @@
-import { has, omit, get } from 'lodash';
+import { get, has, omit } from 'lodash';
```

Comment:
We can revert this change, unless you really care about it.

Reply:
I don't, I was sure it was prettier at first I can't find a rule that does that. Reverting